superset-frontend/src/CRUD/CollectionTable.tsx

```diff
@@ -261,7 +261,7 @@ export default class CRUDCollection extends React.PureComponent<
             aria-label="Delete item"
             className="pointer"
             data-test="crud-delete-icon"
-            iconSize="m"
+            css={{fontSize:"18px"}}
```

Comment:
Can you try using `style` instead of `css` here? I think that's probably the best way to change this, check out this [article](https://stackoverflow.com/questions/54456956/how-can-i-set-the-size-of-icons-in-ant-design).

Reply:
The other thing you could try is to change iconSize to "s", did you try that already?

Reply:
Thanks, @lyndsiWilliams and @AAfghahi! I Changed `css` to `style`. Based on the article you linked, Lyndsi, it looks like I was on the right track with using `fontSize`. I did try setting `iconSize="l"`, but that only brought the icon up to 16px*16px and the ticket said 18px*18px.

Reply:
Perfect, this looks great!